_includes/navbar-w-links.html

```diff
@@ -0,0 +1,46 @@
+<!-- NAV -->
+
+<nav class="navbar">
+	<button onClick="toggleMenu()" class="menu-icon"><img src="/images/hamburger-open-menu.svg" alt="open menu"></button>
```

Comment:
"Open Menu" remains the same whether it's a hamburger icon or an x icon
I also suggest we add an aria-label to the button instead and add aria-hidden to the img, and use aria-expanded to communicate if the menu is open or closed